packages/commonwealth/client/scripts/views/components/AuthButton/AuthButton.tsx

```diff
@@ -0,0 +1,45 @@
+import clsx from 'clsx';
+import React from 'react';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWCustomIcon } from '../component_kit/cw_icons/cw_custom_icon';
+import { CWIcon } from '../component_kit/cw_icons/cw_icon';
+import { CWTag } from '../component_kit/new_designs/CWTag';
+import './AuthButton.scss';
+import { AUTH_TYPES } from './constants';
+import { AuthButtonProps } from './types';
+
+const AuthButton = ({
```

Comment:
wondering if this should be CW component 🤔

Reply:
Tbh I am not sure. I think we started to overuse CW. My opinion right now is that CW should be our building blocks from [design system](https://www.figma.com/file/eIVp33a1oCu0AtcLwSbGjr/%F0%9F%9A%A7-Components-and-Patterns?type=design&node-id=0-1&mode=design&t=J3M4mKuYL837U2oh-0). So buttons, drawers, radios etc. But yeah, hard to define clean boundary. Would love to hear opinion from @Miaplacidus

Reply:
> My opinion right now is that CW should be our building blocks from [design system](https://www.figma.com/file/eIVp33a1oCu0AtcLwSbGjr/%F0%9F%9A%A7-Components-and-Patterns?type=design&node-id=0-1&mode=design&t=J3M4mKuYL837U2oh-0).

i also agree on this. Curious to hear from Ifu.